lib/client/websocket-tracker.js

```diff
@@ -89,6 +89,15 @@ WebSocketTracker.prototype.destroy = function (cb) {
   clearInterval(self.interval)
   clearTimeout(self.reconnectTimer)
 
+  // Destroy peers
+  for(var peerId in self.peers) {
+    var peer = self.peers[peerId]
+    clearTimeout(peer.trackerTimeout)
+    peer.destroy()
+  }
+  delete self.peers
```

Comment:
`self.peers = null` is better. `delete` is for when we want to remove a key from a raw object. `self` is an actual instance of a class, so `delete` is a worse choice. It will still work but it causes a de-optimization in most JS engines.